jena-text/src/main/java/org/apache/jena/query/text/assembler/TextVocab.java

```diff
@@ -78,17 +78,16 @@
     public static final Resource whitespaceTokenizer = Vocab.resource(NS, "WhitespaceTokenizer");
     
     // Filters
+	/** 
+	 * "Standard Filter" is a no-op in Lucene 7.x and removed in Lucene 8.x.
+	 * @deprecated Do not use. To be removed.
+	 */
```

Comment:
tabs? or maybe github UI is not showing spaces correctly.

Reply:
Fixed - must have been some tabs in the file. Eclipse tries to copy the local indent style rather than apply the user settings regardless. It gets confused, not unreasonably, if the original context is then removed.